docs/authentication/README.md

```diff
@@ -0,0 +1,119 @@
+# Authentication Commands
+
+## `pingcli auth`
+
+Main authentication command group for managing CLI authentication with PingOne services.
+
+### Usage
+```bash
+pingcli auth [command]
+```
+
+### Available Commands
+- [`login`](login.md) - Authenticate using OAuth2 flows
+- [`logout`](logout.md) - Clear stored authentication tokens
+
+### Flags
+- `-h, --help` - Help for auth command
+
+### Examples
+```bash
+# View auth help
+pingcli auth --help
+
+# Login with device code flow
+pingcli auth login --device-code
+
+# Logout and clear tokens
+pingcli auth logout
+```
+
+## Quick Start
+
+1. **Configure authentication settings**:
+   ```bash
+   pingcli config set service.pingone.regionCode=NA
+   pingcli config set service.pingone.authentication.deviceCode.clientID=<your-client-id>
+   pingcli config set service.pingone.authentication.deviceCode.environmentID=<your-env-id>
+   ```
+
+2. **Authenticate**:
+   ```bash
+   pingcli auth login --device-code
+   ```
+
+3. **Use authenticated commands**:
+   ```bash
+   pingcli request get /environments
+   ```
+
+4. **Logout when done**:
+   ```bash
+   pingcli auth logout
+   ```
+
+## Technical Architecture
+
+### Token Storage
+
+pingcli uses a **dual storage system** to ensure tokens are accessible across different environments:
+
+1. **Primary Storage**: Secure platform credential stores (via [`pingone-go-client`](https://github.com/pingidentity/pingone-go-client) SDK)
+   - **macOS**: Keychain Services
+   - **Windows**: Windows Credential Manager  
+   - **Linux**: Secret Service API
+
+2. **Secondary Storage**: Encrypted file-based storage at `~/.pingcli/credentials/`
```

Comment:
Is this actually encrypted?

Are we concerned that we might be writing credentials to files without users being aware? Wondering if this would need to be opt-in.

Reply:
They can set the new file storage flag to false

Reply:
Various documents still falsely list these as encrypted